examples/data/.gitignore

```diff
@@ -0,0 +1 @@
+output
```

Comment:
This is interesting, I guess you ned to ignore some output from the examples (but I can't find where)? 
I would expect this at the root of the module then.

Reply:
The _CustomRuleSample4_ if run with the parameters from `README.md`creates the files.
But you bringing this to attention, I wonder if it's needed all that much (doubting my own decision 😉 )
On general note though - you would prefer the subdirectory to be ignored form the module root instead of locally?

Reply:
I think a `.gitignore` file belongs at the root of a project. If for whatever reason it's necessary to specifically ignore something int a submodule, that should ideally be in the general `.gitignore` pointing to the submodule or at the root of the specifc module. I've never seen `.gitignore` being placed in other places, I don't even know if that works. I know I'd have a hard time finding out why something is ignored if it's not in the root-level `.gitignore`

Reply:
Removing the file (forgot to do it in previous commits)